zarp/run/snakemake.py

```diff
@@ -0,0 +1,49 @@
+"""Module for running snakemake."""
+
+import subprocess
+import os
+
+from zarp.config.models import Run
+
+
+class SnakemakeExecutor:
+    """Run snakemake with system calls."""
+
+    def __init__(self, run: Run) -> None:
+        self.run_dict = run.dict()
+        self.success = None
+
+    def prepare_run(self, snkfile: str, workdir: str) -> None:
+        """Configure list of strings for execution.
+        """
+        run_list = ["snakemake"]
+        # Snakemake config
+        run_list.extend(["--snakefile", snkfile])
+        run_list.extend(["--cores", str(self.run_dict["cores"])])
+        # run_list.extend(["--config", f"workdir={workdir}"])
```

Comment:
Make sure to remove or uncomment such comments in your final PR

Reply:
Right now, no workdir is set, so I possibly still need this, but since I did not test it, it's uncommented.
But sure, I try to remove all spurious code for the final PR.